### DISCUSSION TITLE
Fed Ex Freight Support

### DISCUSSION THREAD

**Discussion by duerz** [alexeybusygin/ShippingRates#30] [Ideas]
I am trying to use your app for Fed Ex freight.  But your api version for fed ex is 26, and I called Fed Ex looking for documentation - and they could not find.  Would you please consider upgrading your project to version 31?

**Comment by joekrauss** [comment]
This has my vote as well. When I try adding a package 48x48x48 120 lb. the rate returns the following message. 

Unknown FedEx rate code: FEDEX_FIRST_FREIGHT
Unknown FedEx rate code: FEDEX_1_DAY_FREIGHT
Unknown FedEx rate code: FEDEX_2_DAY_FREIGHT
Unknown FedEx rate code: FEDEX_3_DAY_FREIGHT

**Comment by alexeybusygin** [comment]
Hi! I upgraded FedEx to version 31. But freight shipping is a bit different and will require some additional work. But there were so many requests to implement it so I put it in work.

**Comment by alexeybusygin** [comment]
FedEx Freight support is available now in version 1.1.9.
